src/app/components/Item/ItemTableRow.jsx

```diff
@@ -42,10 +42,10 @@ const ItemTableRow = ({ item, bibId, getRecord, searchKeywords }) => {
 
   return (
     <tr className={item.availability}>
-      <td>{item.location}</td>
-      <td>{itemCallNumber}</td>
-      <td>{itemRequestBtn}</td>
-      <td>{item.accessMessage.prefLabel}</td>
+      <td>{item.location || ' '}</td>
+      <td>{itemCallNumber || ' '}</td>
```

Comment:
oops, I see what you are saying.
But we can still have this logic in the check from line 39 - 41. Since we don't want `''` at all but `' '`.